src/main/drivers/display_ug2864hsweg01.c

```diff
@@ -274,6 +293,43 @@ void i2c_OLED_send_string(busDevice_t *bus, const char *string)
     }
 }
 
+void i2c_OLED_send_string_vertical(busDevice_t *bus, uint8_t x, uint8_t y, const char *string)
+{
+    // vertical mode
+    i2c_OLED_set_vertical_adressing_mode(bus);
+
+    uint8_t buffer[SCREEN_HEIGHT/8];
+
+    // Sends a string of chars until null terminator
+    const char *sptr;
+    uint8_t i;
+    uint8_t j;
+
+    uint8_t len = strlen(string);
+    if (len > SCREEN_HEIGHT/8) len = SCREEN_HEIGHT/8;
+
+    for (i = 0; i < 5; i++) {
+        i2c_OLED_set_xy(bus, x + i, y);
+        sptr = string;
+        for(j = 0; j<len; j++) {
+            buffer[j] = multiWiiFont[*sptr - 32][i];
+            buffer[j] ^= CHAR_FORMAT;  // apply
+            sptr++;
+        }
+        i2cWriteBuffer(bus->busdev_u.i2c.device, bus->busdev_u.i2c.address, 0x40, len, buffer);
+    }
+    // send blank line after string:
+    for(j = 0; j<len; j++) {
+        buffer[j] = 0;
```

Comment:
`buffer[j] = CHAR_FORMAT;` to match  `i2c_OLED_send_char` / handle inversion

`memset(buffer, CHAR_FORMAT, len);`